Source/Features/Store/NotificationStore.swift

```diff
@@ -190,26 +190,26 @@ public class NotificationStore: Collection, StoreRealTimeObserver {
     /// - Parameters:
     ///    - notification: Notification will be removed.
     ///    - completion: Closure with a `Error`. Success if error is nil.
-    public func delete(_ notification: Notification, completion: @escaping (Error?) -> Void) {
+    public func delete(_ notification: Notification, completion: @escaping (Result<Void, Error>) -> Void) {
```

Comment:
Why this change? if there is no return (Void), just return an optional error.

cc @jfilowk